doc/conf.py

```diff
@@ -630,6 +630,7 @@ def setup(app):
     "https://microsoft.com/",
     "https://www.jstor.org/stable/2984099",
     "https://stat.uw.edu/sites/default/files/files/reports/2000/tr371.pdf",
+    "https://devguide.python.org/triage/triaging/index.html#becoming-a-member-of-the-python-triage-team",
```

Comment:
I think the issue here is actually the anchor `#becoming-a-member-of-the-python-triage-team` I have updated the issue with more info about each "broken" link.

Reply:
Thanks, will do now. I found that was the case with the Google Open source link.  My commit for that particular link will be to the corresponding .rst (about.rst)

Reply:
Not sure what to do here and which section it was pointing to and what we wanted to say with this link.

I agree with @adrinjalali that we should try to find an appropriate link in this case rather than add it to the ignore list.